src/commands/runtime/activation/list.js

```diff
@@ -103,26 +144,46 @@ class ActivationList extends RuntimeBaseCommand {
               }
             }
           },
-          Duration: {
-            get: row => row.duration ? `${row.duration}ms` : '--'
+          Wait: {
+            get: (row) => {
+              if (row.duration !== undefined) {
+                const { annotations } = row
+                if (!annotations || !annotations.length) return
+                const elem = annotations.find((elem) => {
+                  return (elem.key === 'waitTime')
+                })
+                return elem ? elem.value : '--'
+              } else {
+                return '--'
+              }
+            }
           },
-          Status: {
+          Init: {
             get: (row) => {
-              const statusStrings = ['success', 'application error', 'developer error', 'internal error']
```

Comment:
@rabbah  We make this distinction between application error (any error from within the action) and developer error (eg:- failed to start action) so that it's easy for users to debug. Can you explain a little about your change here. It appears that we are loosing this distinction atleast in UI.

Reply:
Do you mean that the distinction is lost because of the presence of “internal error” which created a third error category?

(internal error represents a system failure usually outside the users control)

Reply:
I think `internal error` is still fine because it has been replaced it with `syserror`. My question was about `'application error', 'developer error'` which is now replaced with `'error', 'error${decorators.collision}'`. Activations returning statusCode of 1 and 2 currently would show as having an `application error` and `developer error` respectively. With your change, we would only see `error` IIUC ?

Reply:
Understood. There are actually three different values when the status code is 1 or 2:

error
error 💥 
timeout

I can factor the strings emitted for the status code into its own function which can be overriden easily so aio may use one version an downstream clients may tailor as needed. 

I’m open to suggestions.

Reply:
Yes that would be a good way to go @rabbah . Also we would still need the feature toggle for now. This is because we don't have a major release planned for a while.

Reply:
Yep - I will incorporate a feature flag soon.